about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="style.css">
+    <title>About</title>
+</head>
+<body id = 'about'>
+     <!-- first page for about page -->
+    <section>
+        <nav class="nav-top-desktop">
+            <ul>
+                <li><i class="fa-brands fa-facebook-f"></i></li>
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </nav>
+        <div class="nav-top-mobile">
+            <button id="wrapper-about"><i class="fa-sharp fa-solid fa-bars"></i></button>
+        </div>
+        <div class="nav-wrapper" id="nav-wrapper-about">
+            <a class="close-button" id="close-wrap-about"><i class="fa-solid fa-xmark"></i></a>
+            <div>
+                <a href="index.html">Home</a>
+                <a href="">About</a>
+                <a href="">Program</a>
+                <a href="">Join</a>
+                <a href="">Sponsor</a>
+                <a href="">News</a>
+                <a href="">IOT Campaign</a>
+            </div>
+        </div>
+        <div class="nav-wrapper-desktop">
+            <h1>IOT-Horizons</h1>
+            <div>
+                <a href="index.html">Home</a>
+                <a href="">Program</a>
+                <a href="">Join</a>
+                <a href="">Sponsor</a>
+                <a href="">News</a>
+                <a href="">IOT Campaign</a>
+            </div>
+        </div>
+        <div class="presentation-container-about">
+            <header>
+                <h3>"Hello Tech world"</h3>
+                <h1>IOT CONGRESS 2023</h1>
+            </header>
+            <h6 class="line">────</h6>
+            <p class="main-description-about">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Sed, facere? Non blanditiis eligendi eaque adipisci consectetur expedita, nobis quasi tenetur dolore, officia quas fuga ratione debitis dolores, molestias possimus illum.</p>
+            <div class="presentation-container-footer">
```

Comment:
## The same for about page 💯💯
- [ ] I think it would be great if you add a real text instead of lorem ipsum since you should personalize your content since to have a real project It's very important the you personalize your project so that you have something unique in your portfolio to share with potential employers during job searching in the future . so please personalize it ⭐♻💯🚀🚀


![image](https://user-images.githubusercontent.com/58553711/219302045-44351094-9f1d-43c6-9b10-a0bc9f9375db.png)